index.bs

```diff
@@ -40,16 +40,14 @@ protections have been shipped and / or are planned. This section is not
 comprehensive.
 
 <h3 id="mitigations-safari">Safari</h3>
-Safari uses an algorithmic approach to combat navigational-tracking. Safari
+Safari uses an algorithmic approach to combat navigational tracking. Safari
 labels a site as having navigational-based cross-site tracking capability if
 the following criteria are met within a particular client:
 
-<ul>
-  <li>The site has not served content as a third-party.</li>
-  <li>The site automatically redirects the user to another site, immediately
-      or after a short delay.</li>
-  <li>The site has not received a user activation.</li>
-</ul>
+* The site has not served content as a third-party.
+* The site automatically redirects the user to another site, immediately
```

Comment:
The site automatically redirects the user to other sites, …
I forget the exact minimum fan-out number but I think it's three. If a site redirects to a minimum of three other sites it shows that it's a dispatch of some sort.